docs/release-notes/job-queued-stats.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  Fix an issue where job queued stats for jobs submitted to k8s would incorrectly show a very high
+   amount.
```

Comment:
:orphan:

**Bug Fixes**

-  Fix an issue where jobs submitted to Kubernetes would incorrectly display an excessively high queue count.